src/RA_StringUtils.h

```diff
@@ -329,54 +329,77 @@ class StringBuilder
         while (*pFormat)
         {
             auto* pScan = pFormat;
+            Expects(pScan != nullptr);
```

Comment:
`T` couldn't be deduced, even explicitly, so we had to check for nullness on all paths.